reconnect_conn.go

```diff
@@ -0,0 +1,148 @@
+package sloglogstash
+
+import (
+	"errors"
+	"net"
+	"sync"
+	"sync/atomic"
+	"time"
+)
+
```

Comment:
Can you add this please:

```go
var _ net.Conn = (*ReconnectConn)(nil)
```

It will check on build, that ReconnectConn implements net.Conn.

Reply:
Added, thanks for the explanation, I wondered why you used that